Assets/Scripts/Inventory/ItemPresenter.cs

```diff
@@ -8,6 +8,7 @@ public class ItemPresenter : MonoBehaviour
     {
         [SerializeField] private Image itemImage;
         // TODO: [SerializeField] private Text amountText;
+        [SerializeField] private Sprite emptySlotImage;
```

Comment:
since the emptySlotImage for all ItemPresenters is the same, we dont need to use this serializedField, loading from resources would do